wallet-new/test/unit/UTxO/DSL.hs

```diff
@@ -354,12 +359,21 @@ utxoToList = Map.toList . utxoToMap
 utxoDomain :: Utxo h a -> Set (Input h a)
 utxoDomain = Map.keysSet . utxoToMap
 
+utxoRange :: Utxo h a -> [Output a]
```

Comment:
Are 'Range' and 'Domain' conventionally used in the context of UTXO accounting? Otherwise these aren't terribly obvious unless you're aware of the Map inside `Utxo`, which somewhat breaks the abstraction.

Reply:
This is terminology straight from the formal model.